### DISCUSSION TITLE
Alternatives to Zentity

### DISCUSSION THREAD

**Discussion by prabhushan** [zentity-io/zentity#120] [General]
Hi all,
Since there was no updates in Zentity to support elasticsearch 8.X , what other alterative solution we have - Commerical / Opensource?

**Comment by davemoore-** [comment]
Note: [zentity-1.8.3](https://zentity.io/releases/#zentity-1.8.3) now supports Elasticsearch 8.13 (more details [here](https://github.com/zentity-io/zentity/issues/108#issuecomment-2093777549))
